Java/src/main/java/Runner.java

```diff
@@ -1,7 +1,7 @@
 import models.MilitaryType;
```

Comment:
Please create "runner" package and move this class to it

Reply:
Created and moved